bindings/core/src/message_interface_old/account_method.rs

```diff
@@ -48,6 +48,8 @@ pub enum AccountMethod {
     BuildAccountOutput {
         // If not provided, minimum storage deposit will be used
         amount: Option<String>,
+        #[serde(with = "string")]
+        mana: u64,
```

Comment:
Why is that different from the amount?

Reply:
I mean, why not a String

Reply:
It's better to use the real type and let serde manage the conversions, but I didn't want to also re-implement all my changes from #893 in this PR.